pygeos/test/test_geometry.py

```diff
@@ -21,18 +22,20 @@
 
 
 def test_get_num_points():
-    actual = pygeos.get_num_points(all_types).tolist()
-    assert actual == [0, 3, 5, 0, 0, 0, 0, 0, 0]
+    actual = pygeos.get_num_points(all_types + (None,)).tolist()
+    assert actual == [0, 3, 5, 0, 0, 0, 0, 0, 0, 0]
 
 
 def test_get_num_interior_rings():
-    actual = pygeos.get_num_interior_rings(all_types + (polygon_with_hole,)).tolist()
-    assert actual == [0, 0, 0, 0, 0, 0, 0, 0, 0, 1]
+    actual = pygeos.get_num_interior_rings(
+        all_types + (polygon_with_hole, None)
+    ).tolist()
+    assert actual == [0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0]
```

Comment:
```suggestion
    actual = pygeos.get_num_interior_rings(all_types + (polygon_with_hole, None))
    assert actual.tolist() == [0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0]
```

(slightly more readable IMO)